tests/integration/cmd_project_test.go

```diff
@@ -40,14 +40,12 @@ var _ = Describe("odo project command tests", func() {
 
 	Context("when running project command app parameter in directory that doesn't contain .odo config directory", func() {
 		It("should successfully execute list along with machine readable output", func() {
-			helper.CmdRunner("odo", "project", "list")
-			time.Sleep(1 * time.Second)
 			listOutput := helper.CmdShouldPass("odo", "project", "list")
 			Expect(listOutput).To(ContainSubstring(project))
 
 			// project deletion doesn't happen immediately, so we test subset of the string
 			listOutputJson := helper.CmdShouldPass("odo", "project", "list", "-o", "json")
-			Expect(listOutputJson).To(ContainSubstring(`{"kind":"Project","apiVersion":"odo.openshift.io/v1alpha1","metadata":{"name":"` + project + `","namespace":"` + project + `","creationTimestamp":null},"spec":{},"status":{"active":false}}`))
+			Expect(listOutputJson).To(ContainSubstring(`{"kind":"Project","apiVersion":"odo.openshift.io/v1alpha1","metadata":{"name":"` + project + `","namespace":"` + project + `","creationTimestamp":null},"spec":{},`))
```

Comment:
Shouldn't this be status: active? Why was it removed? This is obvious that the test is failing.. In this test scenario, you can modify the `beforeEach` to add `-w` to wait for the project to come up.

Reply:
> Shouldn't this be status: active? Why was it removed? This is obvious that the test is failing..

@cdrage https://github.com/openshift/odo/pull/2221#discussion_r330991328

Think a case where you have more scenarios are running for example ```make test-integration``` then in two node you cannot be sure that the current scenario project is active throughout the scenario. May be in the same time some different project scenario is created and become active.

> In this test scenario, you can modify the beforeEach to add -w to wait for the project to come up

In the test script we have already made this behavior as default to avoid flakes. see - https://github.com/openshift/odo/blob/master/tests/helper/odo_utils.go#L79. The failure was happening due to the step ```CmdRunner(...)``` which does not wait for command output.

Reply:
As @amitkrout pointed due to parallel tests we can't be sure of the activeness of the test throughout the scenario so I removed the status. @cdrage @kadel WDYT?

Reply:
Ah, but that defeats the whole purpose of the test though, to make sure that it's the active project.

Reply:
yup, there is an option of sequential run. In a long run it would be hard to even manage ```get``` and ```set``` command in parallel run. @mik-dass Please do the change to run it on single node test.

For reference check - https://github.com/openshift/odo/tree/master/tests/integration/loginlogout.